### ISSUE TITLE
Error in wide-resnet implementation

### ISSUE THREAD

**Issue by jpizarrom** [keras-team/keras-contrib#487]
I think the wide rest net is wrong
I've tried 
```
model = WideResidualNetwork(depth=16, width=1, dropout_rate=0.0, weights=None)
```
and I got [current] , but I think should be [correct]
## current
![model keras_contrib applications wide_resnet](https://user-images.githubusercontent.com/111236/55637620-c235f280-57c5-11e9-9040-95803fb5dc57.png)

## correct
![model](https://user-images.githubusercontent.com/111236/55637623-c3ffb600-57c5-11e9-8fc8-14633c73eed2.png)

**Comment by jpizarrom** [comment]
I've plotted the mode from the repo of the paper https://arxiv.org/abs/1605.07146
```
import torch
from torchviz import make_dot, make_dot_from_trace

from resnet import resnet as pytorch_resnet
f, params = pytorch_resnet(16, 1, 10)
x = torch.rand(1, 3, 32, 32).cuda()
dot = make_dot(f(x, params, mode=False), params=dict(params))
dot.format = 'png'
dot.render('pytorch_resnet')  
```

I've got 
![pytorch_resnet](https://user-images.githubusercontent.com/111236/55650408-e86b8a80-57e5-11e9-85eb-dc51c51d9111.png)

**Comment by titu1994** [comment]
The first block of your correction has two consecutive batchnorm-relu blocks. 

This codebase has also been here for nearly two gears without significant updated. If you perceive an issue during training, can you plot the training curves of both the old model and your proposed correction and display the merit of an update.

**Comment by jpizarrom** [comment]
I'm currently working in the model, I will fix my issue with "two consecutive batchnorm-relu blocks."

I will try to plot both curves in the next days

thanks

**Comment by jpizarrom** [comment]
> The first block of your correction has two consecutive batchnorm-relu blocks.
> 
> This codebase has also been here for nearly two years without significant updated. If you perceive an issue during training, can you plot the training curves of both the old model and your proposed correction and display the merit of an update.

I've fixed and update the mode diagram in the description

Thanks

**Comment by titu1994** [comment]
Sure if after running a comparison on CIFAR 10/100 we find that the older model performs significantly worse, then you could submit a PR with corrections. 

Thanks for taking a look at this.
